presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeTpcdsQueries.java

```diff
@@ -0,0 +1,1157 @@
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("64"));
+    }
+
+    @Test
+    public void testTpcdsQ65()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("65"));
+    }
+
+    @Test
+    public void testTpcdsQ66()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("66"));
+    }
+
+    @Test
+    public void testTpcdsQ67()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("67"));
+    }
+
+    @Test
+    public void testTpcdsQ68()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("68"));
+    }
+
+    @Test
+    public void testTpcdsQ69()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("69"));
+    }
+
+    @Test
+    public void testTpcdsQ70()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("70"));
+    }
+
+    @Test
+    public void testTpcdsQ71()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("71"));
+    }
+
+    @Test
+    public void testTpcdsQ72()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("72"));
+    }
+
+    @Test
+    public void testTpcdsQ73()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("73"));
+    }
+
+    @Test
+    public void testTpcdsQ74()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("74"));
+    }
+
+    @Test
+    public void testTpcdsQ75()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("75"));
+    }
+
+    @Test
+    public void testTpcdsQ76()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("76"));
+    }
+
+    @Test
+    public void testTpcdsQ77()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("77"));
+    }
+
+    @Test
+    public void testTpcdsQ78()
+            throws Exception
+    {
+        assertQueryFails(session, getTpcdsQuery("78"), "[\\s\\S]*Scalar function presto\\.default\\.round not registered with arguments[\\s\\S]*");
+    }
+
+    @Test
+    public void testTpcdsQ79()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("79"));
+    }
+
+    @Test
+    public void testTpcdsQ80()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("80"));
+    }
+
+    @Test
+    public void testTpcdsQ81()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("81"));
+    }
+
+    @Test
+    public void testTpcdsQ82()
+            throws Exception
+    {
+        assertQuery(session, getTpcdsQuery("82"));
+    }
+
+    @Test
+    public void testTpcdsQ83()
+            throws Exception
+    {
+        assertQueryFails(session, getTpcdsQuery("83"), "[\\s\\S]*Filter\\(MultiRange, deterministic, null not allowed\\): testInt64\\(\\) is not supported[\\s\\S]*");
```

Comment:
https://github.com/prestodb/presto/pull/19593 was merged yesterday. This should fix this query.

Reply:
@aditi-pandit This was added before the change. Removed after rebase